core/modules/update/theme/update.theme.inc

```diff
@@ -78,17 +78,21 @@ function theme_update_report($variables) {
   }
 
   foreach ($data as $project) {
+    // Don't show a link to the "Update modules" page for core.
+    $update_link = ($project['name'] != 'backdrop');
     switch ($project['status']) {
       case UPDATE_CURRENT:
         $class = 'status';
         $icon = theme('image', array('uri' => 'core/misc/watchdog-ok.png', 'width' => 18, 'height' => 18, 'alt' => t('ok'), 'title' => t('ok')));
+        $update_link = 0;
         break;
       case UPDATE_UNKNOWN:
       case UPDATE_NOT_CHECKED:
       case UPDATE_FETCH_PENDING:
       case UPDATE_NOT_FETCHED:
         $class = 'unknown';
         $icon = '';
+        $updatelink = 0;
```

Comment:
@laryn this needs to be `$update_link` with the underscore.   Up for another commit?